sample/src/androidTest/java/com/adevinta/android/barista/sample/util/MaterialAttributes.java

```diff
@@ -0,0 +1,6 @@
+package com.adevinta.android.barista.sample.util;
+
+public interface MaterialAttributes {
```

Comment:
Because project is using Java, we could not use alias, but I also wanted to avoid inlined packages

Reply:
Explanation: latest gradle enabled by default `nonTransitiveRClass`